circle.yml

```diff
@@ -0,0 +1,212 @@
+defaults: &defaults
+  docker:
+    - image: circleci/node:8-browsers
+      environment:
+        JOBS: 2
+
+node_key: &node_key
+  key: node-deps-{{ checksum "yarn.lock" }}-{{ checksum "circle.yml" }}
+
+restore_node: &restore_node
+  restore_cache:
+    <<: *node_key
+
+deploy_static: &deploy_static
+  <<: *defaults
+  steps:
+    - checkout
+    - <<: *restore_node
+    - deploy:
+        name: Deploy to S3
+        command: |
+          env | sed -n "s/^${ENV}_//p" > .env
+          npx ember deploy `echo $ENV | tr A-Z a-z` --verbose --activate
+
+deploy_fastboot: &deploy_fastboot
+  docker:
+    - image: circleci/python:3.6
+  steps:
+    - checkout
+    - setup_remote_docker
+    - restore_cache:
+        key: deploy-{{ checksum "circle.yml" }}
+    - run:
+        name: Install Requirements
+        command: |
+          if [[ ! -d ~/.venv ]]; then
+            python3 -m venv ~/.venv
+            source ~/.venv/bin/activate
+            pip3 install -U git+https://github.com/nypublicradio/nyprsetuptools.git
+          fi
+    - deploy:
+        name: Deploy Fastboot
+        command: |
+          TAG=${CIRCLE_TAG:-demo}
+          source ~/.venv/bin/activate
+          nyprsetuptools DockerDeploy --environment=$ENV \
+                                      --ecs-cluster=fastboot \
+                                      --ecr-repository=nypr-jukeboxes \
+                                      --tag=$TAG \
+                                      --wait=300 \
+                                      --fargate \
+                                      --execution-role=nypr-jukeboxes-$ENV \
+                                      --task-role=nypr-jukeboxes-$ENV \
+                                      --ports=3000 \
+                                      --cpu=256 \
+                                      --memory-reservation=512
+    - save_cache:
+        key: deploy-{{ checksum "circle.yml" }}
+        paths:
+          - ~/.venv
+
+version: 2
+
+jobs:
+  build-static:
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      - <<: *restore_node
+      - run:
+          name: Install node dependencies
+          command: |
+            if [ ! -d node_modules ]; then
+              yarn --pure-lockfile
+            fi
+      - save_cache:
+          <<: *node_key
+          paths:
+            - node_modules
+            - /home/circleci/.cache/Cypress
+
+  test-static:
+    <<: *defaults
+
+    steps:
+      - checkout
+      - <<: *restore_node
+      - run:
+          name: Test
+          command: npx ember test
+
+  test-static-cypress:
```

Comment:
I think you can comment this out for now – Cypress isn't set up for this yet

Reply:
Do I comment out the whole `save_cache` step starting line 78?

Reply:
Just comment out or remove lines 94-105